Documentation/introduction/resources.rst

```diff
@@ -9,7 +9,7 @@ Here's a list of Apache NuttX resources that you might find helpful:
  * Apache NuttX
 
    * `NuttX website <https://nuttx.apache.org>`_
-   * `Apache NuttX website <https://www.nuttx.org>`_
+   * `Apache NuttX website <https://apache.nuttx.org>`_
    * `Apache NuttX online documentation <https://cwiki.apache.org/confluence/display/NUTTX/Nuttx>`_
    * `Apache NuttX mailing list <https://nuttx.incubator.apache.org/community/>`_ – a very active mailing list, the place to get help with your application or any questions you have about NuttX.
```

Comment:
Note that `nuttx.incubator.apache.org` is used here. Maybe you can also sed that term.